README.md

```diff
@@ -1,12 +1,10 @@
 # Under development 
-This package is still in the early development phase. 
-# MetopNative.jl
-[![Build Status](https://gitlab.eumetsat.int/eumetlab/cross-cutting-tools/MetopNative.jl/badges/main/pipeline.svg)](https://gitlab.eumetsat.int/eumetlab/cross-cutting-tools/MetopNative.jl/pipelines)
-[![Coverage](https://gitlab.eumetsat.int/eumetlab/cross-cutting-tools/MetopNative.jl/badges/main/coverage.svg)](https://gitlab.eumetsat.int/eumetlab/cross-cutting-tools/MetopNative.jl/commits/main)
+This package is still in the early development phase. Note that the package was previously name **MetopNative.jl** and was hosted on the [EUMETSAT GitLab](https://gitlab.eumetsat.int/eumetlab/cross-cutting-tools/MetopNative.jl) 
+# MetopDatasets.jl
 
-MetopNative.jl is a package for reading products from the [METOP satellites](https://www.eumetsat.int/our-satellites/metop-series) using the native binary format specified for each product. The METOP satellites are part of the EUMETSAT-POLAR-SYSTEM (EPS) and produces long series of near-realtime weather and climate observation. Learn more and access the products on [EUMETSATs user-portal](https://user.eumetsat.int/dashboard).
+MetopDatasets.jl is a package for reading products from the [METOP satellites](https://www.eumetsat.int/our-satellites/metop-series) using the native binary format specified for each product. The METOP satellites are part of the EUMETSAT-POLAR-SYSTEM (EPS) and produces long series of near-realtime weather and climate observation. Learn more and access the products on [EUMETSATs user-portal](https://user.eumetsat.int/dashboard).
```

Comment:
produces -> produce
near-realtime -> near-real-time (?)

Reply:
https://www.eumetsat.int/real-time-imagery/earth-view and https://en.wikipedia.org/wiki/Real-time_computing#:~:text=The%20term%20%22near%20real%2Dtime,or%20feedback%20and%20control%20purposes. say "near real-time" so I will go with that.